modules/core/shared/src/main/scala/lucuma/core/model/SourceProfile.scala

```diff
@@ -61,14 +68,20 @@ object SourceProfile {
 
   }
 
+  /**
+   * Uniform source.
+   *
+   * @param spectralDefinition
+   *   integrated spectral definition
+   */
   final case class Uniform(spectralDefinition: SpectralDefinition[Surface]) extends SourceProfile {
 
     override def toPoint: Point = Point(spectralDefinition.to[Integrated])
 
     override def toUniform: Uniform = this
 
     override def toGaussian: Gaussian =
-      Gaussian(GaussianSource.Zero, spectralDefinition.to[Integrated])
+      Gaussian(Angle.Angle0, spectralDefinition.to[Integrated])
```

Comment:
This seems a bit odd, Would uniform have a width 0?

Reply:
The use case I imagine for this is in the UI having a drop-down for `Source Profile` with options `Point`, `Uniform` and `Gaussian`. If changing from `Point`/`Uniform` -> `Gaussian`, the angle input will appear, with a default value. I set it to `Angle0` but maybe it makes more sense to have another default value here.

This conversion between source profile types can also be used in the API.